docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -34,6 +34,10 @@ The following table shows which browser is used for the various platforms and op
 
 ## Troubleshooting Microsoft Edge Issues
 
+### Chromium-based Edge is installed on my development computer, but my add-in does not use it
+
+The base browser in [Microsoft Edge](https://support.microsoft.com/en-us/help/4501095/download-the-new-microsoft-edge-based-on-chromium) has changed to Chromium. But the older base, called EdgeHTML, is not removed when Chromium-based Edge is installed. Office will still use the EdgeHTML base for add-ins until a build of Office 365 that supports Chromium is installed on the computer. We expect these builds to ship in 2020. They will likely appear in the Insiders channel in the first half of the year.
```

Comment:
The second sentence "But the older base, called EdgeHTML, is not removed when Chromium-based Edge is installed." seems unnecessary for an add-in developer to know. Can we just remove it?

If so, change the next sentence a bit to indicate EdgeHTML is the old version ("Office will still use Edge's old base, EdgeHTML, for add-ins...").

Reply:
@Rick-Kirkham  Fair enough. Can we remove the "But" from the beginning of that sentence ("The older base...")?